ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/AbstractTaskFragment.kt

```diff
@@ -275,6 +276,10 @@ abstract class AbstractTaskFragment<T : AbstractTaskViewModel> : AbstractFragmen
 
   data class ButtonData(val index: Int, val button: TaskButton)
 
+  fun getDateFormatter(): java.text.DateFormat? = DateFormat.getDateFormat(requireContext())
+
+  fun getTimeFormatter(): java.text.DateFormat? = DateFormat.getTimeFormat(requireContext())
```

Comment:
Can we move this helper method to TimeTaskFragment?

Reply:
Done